src/m/Settings.java

```diff
@@ -1,69 +1,125 @@
+ * They are applied first thing, and then updated from the .cfg file.
+ * 
+ * The file now also contains settings on many features introduced by this new version of Mocha Doom
+ *  - Good Sign 2017/04/11
  */
-
 public enum Settings {
-    mouse_sensitivity("5"),
-    sfx_volume("8"),
-   music_volume("8"),
-   show_messages("1"),
-   alwaysrun("0"),	// Always run is OFF
-   key_right(KEY_RIGHTARROW),
-   key_left(KEY_LEFTARROW),
-   key_up('w'),
-   key_down('s'),
-   key_strafeleft('a'),
-   key_straferight('d'),
-   key_fire(KEY_CTRL),
-   key_use(' '),
-   key_strafe(KEY_ALT),
-   key_speed(KEY_SHIFT),
-   use_mouse(1),
-   mouseb_fire(0),
-   mouseb_strafe(2),	// AX: Fixed
-   mouseb_forward(1),	// AX: Value inverted with the one above
-   use_joystick( 0),
-   joyb_fire(0),
-   joyb_strafe(1),
-   joyb_use(3),
-   joyb_speed(2),
-   screenblocks(10),
-   detaillevel(0),
-   snd_channels(32),
-   usegamma(0),
-   mb_used(2),
-   chatmacro0(HUSTR_CHATMACRO0 ),
-   chatmacro1(HUSTR_CHATMACRO1 ),
-   chatmacro2( HUSTR_CHATMACRO2 ),
-   chatmacro3(HUSTR_CHATMACRO3 ),
-   chatmacro4( HUSTR_CHATMACRO4 ),
-   chatmacro5( HUSTR_CHATMACRO5 ),
-   chatmacro6(HUSTR_CHATMACRO6 ),
-   chatmacro7( HUSTR_CHATMACRO7 ),
-   chatmacro8(HUSTR_CHATMACRO8 ),
-   chatmacro9( HUSTR_CHATMACRO9 );
-
-         private Settings(String defaultval){
-            this.value=defaultval;
-        }
-         
-
-         private Settings(int defaultval){
-            this.value=Integer.toString(defaultval);
-        }
-    
-    public String value;
+    automap_plotter_style(Plotter.Style.Thick),
+    mouse_sensitivity(5),
+    sfx_volume(8),
+    music_volume(8),
+    show_messages(true),
+    alwaysrun(false), // Always run is OFF
+    key_right((int) KEY_RIGHTARROW),
+    key_left((int) KEY_LEFTARROW),
+    key_up((int) 'w'),
+    key_down((int) 's'),
+    key_strafeleft((int) 'a'),
+    key_straferight((int) 'd'),
+    key_fire((int) KEY_CTRL),
+    key_use((int) ' '),
+    key_strafe((int) KEY_ALT),
+    key_speed((int) KEY_SHIFT),
+    use_mouse(true),
+    mouseb_fire(0),
+    mouseb_strafe(2), // AX: Fixed
+    mouseb_forward(1), // AX: Value inverted with the one above
+    use_joystick(false),
+    joyb_fire(0),
+    joyb_strafe(1),
+    joyb_use(3),
+    joyb_speed(2),
+    screenblocks(10),
+    detaillevel(0),
+    snd_channels(32),
+    usegamma(0),
+    mb_used(2),
+    chatmacro0(HUSTR_CHATMACRO0),
+    chatmacro1(HUSTR_CHATMACRO1),
+    chatmacro2(HUSTR_CHATMACRO2),
+    chatmacro3(HUSTR_CHATMACRO3),
+    chatmacro4(HUSTR_CHATMACRO4),
+    chatmacro5(HUSTR_CHATMACRO5),
+    chatmacro6(HUSTR_CHATMACRO6),
+    chatmacro7(HUSTR_CHATMACRO7),
+    chatmacro8(HUSTR_CHATMACRO8),
+    chatmacro9(HUSTR_CHATMACRO9),
+    color_depth(BppMode.Indexed),
+    fullscreen(false),
+    fix_gamma_ramp(true),
+    fix_gamma_palette(true),
+    fix_sky_palette(false),
+    fix_medi_need(true),
+    fix_ouch_face(true),
+    line_of_sight(LOS.Vanilla),
+    vestrobe(true),
```

Comment:
Fix medi need must be false.
Fix ouch face must be false too.
vestrobe (the strobing on the automap?) must be false too.